mdx_oembed/endpoints.py

```diff
@@ -6,6 +6,14 @@
     'https?://youtu.be/*',
 ])
 
+SLIDESHARE = oembed.OEmbedEndpoint('http://www.slideshare.net/api/oembed/2', [
+    'http://www.slideshare.net/*/*',
```

Comment:
@rennat what do you say about the regexp. if you want a unified url then lets do that. I am interested in getting this addition in core and if possible a bump in version / push to pip. What do you think